src/test/java/ro/kuberam/libs/java/crypto/encrypt/SymmetricEncryptionTest.java

```diff
@@ -68,21 +63,21 @@ public void aesAlgorithmAndCbcMode() throws Exception {
 	}
 	
 	@Test
-	public void aesAlgorithmAndDefaultProviderAndEcbMode() throws Exception {
+	public void aesAlgorithmAndDefaultProviderAndEcbMode() throws CryptoException, IOException {
 		byte[] encryptionResult = SymmetricEncryption.encrypt(longInputBytes, key, aesAlgorithmEcbMode, "", "");
 		byte[] decryptionResult = SymmetricEncryption.decrypt(encryptionResult, key, aesAlgorithmEcbMode, "", "");
 
 		assertEquals(longString, new String(decryptionResult, UTF_8));
 	}
 
-	@Ignore
+	//@Ignore
```

Comment:
This test seems to pass so we can safely remove this, right?

Reply:
Yup.